examples/corgi_adventure/README.md

```diff
@@ -0,0 +1,8 @@
+A simple text-based game in the style of 80s adventure games, in which you play
+a hungry corgi.
+
+How to run:
+
+```
+OPENAI_API_KEY=<your key> python application.py
+```
```

Comment:
Mind generating an image of the state machine? Nice to have in the README. Can just commit the one you generated in `application.py`

Reply:
Also good to add it to this: https://github.com/DAGWorks-Inc/burr/blob/main/docs/examples/simple.rst